Author: Alastair McKinstry <mckinstry@debian.org>ยง
Description: compile fix for shadowed function definitions
 goto-cc compiler says multiple 'usage' functions shadow each other
 due to including 'grib_options' when linking. Declare static.
Bug-Upstream: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=752732
Last-Updated: 2017-10-03
Forwarded: no

--- a/tools/grib_tools.h
+++ b/tools/grib_tools.h
@@ -193,7 +193,7 @@
 const char* grib_options_get_help(const char* id);
 const char* grib_options_get_args(const char* id);
 int grib_options_command_line(const char* id);
-void usage(void);
+void grib_usage(void);
 void usage_doxygen(void);
 int grib_tool_before_getopt(grib_runtime_options* options);
 int grib_tool_init(grib_runtime_options* options);
